src/main/scala/mrpowers/jodie/DeltaHelpers.scala

```diff
@@ -19,6 +19,17 @@ object DeltaHelpers {
       .asInstanceOf[Long]
   }
 
+  def deltaFileSizes(deltaTable: DeltaTable) = {
+    val details: Row = deltaTable.detail().select("numFiles", "sizeInBytes").collect()(0)
+    val (sizeInBytes, numberOfFiles) =
+      (details.getAs[Long]("sizeInBytes"), details.getAs[Long]("numFiles"))
+    Map(
+      "size_in_bytes"              -> sizeInBytes,
+      "number_of_files"            -> numberOfFiles,
+      "average_file_size_in_bytes" -> Math.round(sizeInBytes / numberOfFiles)
```

Comment:
If the table is empty this will crash with ArithmeticException because of the division by zero. Let's handle this to effectively return 0 in that case.

Reply:
I did not handle the division by zero, as I was expecting the no.of files to never be zero but makes sense now